### ISSUE TITLE
Question about PPO2 ratio calculation

### ISSUE THREAD

**Issue by R1ckF** [openai/baselines#652]
Hi all,

Thanks for the great work and the constant updates. I was just browsing some of the comments to the code and stumbled upon the following. 
```
        # Calculate ratio (pi current policy / pi old policy)
        ratio = tf.exp(OLDNEGLOGPAC - neglogpac)

        # Defining Loss = - J is equivalent to max J
        pg_losses = -ADV * ratio

        pg_losses2 = -ADV * tf.clip_by_value(ratio, 1.0 - CLIPRANGE, 1.0 + CLIPRANGE)

        # Final PG loss
        pg_loss = tf.reduce_mean(tf.maximum(pg_losses, pg_losses2))
```
From what I understand `OLDNEGLOGPAC` is the placeholder for the old probabilites and `neglogpac` is the current policy probabilty. To me it seems they should be placed the other way around, so: `ratio = tf.exp(neglogpac - OLDNEGLOGPAC)`

Can someone explain what I am missing?
Thanks in advance!

**Comment by pzhokhov** [comment]
@R1ckF correct. neglogpac and OLDNEGLOGPAC are negative log probabilities of the actions; so the ratio works out to be pi(s,a) / pi_old(s, a) (like explained in https://arxiv.org/pdf/1707.06347.pdf). Also, minimizing - A * ratio is equivalent to maximizing A * ratio ( and A is the advantage estimate)
